### ISSUE TITLE
fatal error: systemd/sd-journal.h: No such file or directory

### ISSUE THREAD

**Issue by marshallford** [containers/libpod#3119]
/kind bug

**Description**
Podman `1.3.0` fails to build from source on Alpine. This seems to be a regression from Podman version `1.2.0`. Not every linux distro has access to the systemd development headers. I'm not an expert but this issue seems similar to https://github.com/rkt/rkt/issues/2866.

**Steps to reproduce the issue:**

1. `git clone --branch v1.3.0 https://github.com/containers/libpod $GOPATH/src/github.com/containers/libpod`
2. `cd $GOPATH/src/github.com/containers/libpod`
3. `make podman`

**Describe the results you received:**

```
Step 12/14 : RUN git clone --branch v1.3.0 https://github.com/containers/libpod $GOPATH/src/github.com/containers/libpod &&     cd $GOPATH/src/github.com/containers/libpod && make podman
 ---> Running in 8d67e9022eaa
Cloning into '/go/src/github.com/containers/libpod'...
Note: checking out '139afa7431593bdfca82c9b7ed877f7e41823f69'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

touch .gopathok
go generate ./cmd/podman/varlink/...
go build -ldflags ' -X github.com/containers/libpod/libpod.gitCommit=139afa7431593bdfca82c9b7ed877f7e41823f69 -X github.com/containers/libpod/libpod.buildInfo=1557799539' -tags "seccomp   ostree selinux  varlink exclude_graphdriver_devicemapper" -o bin/podman github.com/containers/libpod/cmd/podman
# github.com/containers/libpod/vendor/github.com/coreos/go-systemd/sdjournal
vendor/github.com/coreos/go-systemd/sdjournal/journal.go:27:11: fatal error: systemd/sd-journal.h: No such file or directory
 // #include <systemd/sd-journal.h>
           ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:115: podman] Error 2
The command '/bin/sh -c git clone --branch v1.3.0 https://github.com/containers/libpod $GOPATH/src/github.com/containers/libpod &&     cd $GOPATH/src/github.com/containers/libpod && make podman' returned a non-zero code: 2
make: *** [Makefile:17: build] Error 2
```

**Describe the results you expected:**

Successfully build podman binary.

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Container image: `golang:1.12.5-alpine3.9`

**Comment by mheon** [comment]
The libsystemd requirement is gated behind a build tag on Master (since #3083 landed) and will be in tomorrow's 1.3.1 release.

**Comment by marshallford** [comment]
@mheon Thanks, I clearly didn't search well enough.
